### PR TITLE
4211 - Adding collaborators refresh list

### PR DESCRIPTION
**Pull request by WillNigel23**

### CODE REVIEW

File: app/assets/javascripts/plugins/jquery.litebox.js [hunk 1]
```diff
@@ -1,266 +1,270 @@
...
+
+  LiteBox.prototype = {
+    defaults: {
+      url: null,              // URL for loading content
+      hash: null,             // Hash to show litebox
+      content: null,          // Selector for local content element
+      cssclass: null,         // Custom CSS class for the litebox
+      noscroll: true,         // Disable page scroll when open
+      disposable: false,      // Remove from DOM after close
+      noclose: false          // Remove close button
```
Comment (WillNigel23) [inline app/assets/javascripts/plugins/jquery.litebox.js:36]: added `noclose` for litebox prototype

File: app/assets/javascripts/plugins/jquery.litebox.js [hunk 2]
```diff
@@ -1,266 +1,270 @@
...
+
+        // Insert into DOM and show
+        if(this.options.noclose) {
+          this.$container.append(this.$content);
+        } else {
+          // Bind close handlers
+          this.$wrapper.add(this.$closebutton).on('click', $.proxy(this.eventClose, this));
+          this.$container.click(function(e) { e.stopPropagation(); });
+          this.$container.append(this.$closebutton, this.$content);
+        }
```
Comment (WillNigel23) [inline app/assets/javascripts/plugins/jquery.litebox.js:99]: lines 92-99

If noclose option passed, we do not add closebutton

**Comment by WillNigel23** [comment]
The diff on the js seems broken (due to empty spaces I assume?)

I marked with comments the lines that I changed. Sorry in advance :pray: 

@benwbrum @saracarl

**Review by saracarl — CHANGES_REQUESTED** [review]
Can you do the same thing with collection owners?

**Comment by benwbrum** [comment]
This works well.

**Pull request merged**